app/Http/Regions/DestinationParents.php

```diff
@@ -6,14 +6,18 @@ class DestinationParents
 {
     public function render($parents, $short = false)
     {
+        $loopCount = 0;
+        $parentsCount = $parents->count();
+
         return component('Meta')
-            ->with('items', $parents->map(function ($parent) use ($short) {
+            ->with('items', $parents->map(function ($parent) use ($short, &$loopCount, $parentsCount) {
```

Comment:
- Add a comment what does $loopCount do and why we need it
- Lets split it into separate PR **Frontpage / destination speed improvements**